Sources/TokamakCore/MountedViews/MountedHostView.swift

```diff
@@ -57,7 +60,9 @@ public final class MountedHostView<R: Renderer>: MountedElement<R> {
     guard !view.children.isEmpty else { return }
 
     mountedChildren = view.children.map {
-      $0.makeMountedView(reconciler.renderer, target, environmentValues, self)
+      let child = $0.makeMountedView(reconciler.renderer, target, environmentValues, self)
+      child.viewTraits = self.viewTraits
+      return child
```

Comment:
Since `viewTraits` is being set immediately, it would make sense to let `makeMountedView` set it directly.